src/main/java/de/retest/recheck/ui/descriptors/Element.java

```diff
@@ -213,6 +213,21 @@ public Attributes getAttributes() {
 		return attributes;
 	}
 
+	/**
+	 * Returns the value of the given attribute, whether this is an identifying attribute or not.
+	 *
+	 * @return The {@link java.io.Serializable} value of the attribute.
+	 */
+	public Object getAttributeValue( final String attributeName ) {
```

Comment:
When it's `Serializable` as stated in the Javadoc, does it make sense to adapt the return type accordingly?

Reply:
Or even generic `T extends Serializable` or simply `T`.to avoid casting at ten different places.

Reply:
There is no common return type `Serializable`. Adapting that would be out of scope of this PR. I do not understand how I could return a generic here...